// @skillsoft-security-chapter/percipiosecretloader/docs/example3.template.config.js
module.exports = {
  account_portal_v2_bff_cognito_client_id: "${ secrets.getValue('account-portal-v2-bff-cognito_client_id-awsdev') }",
  account_portal_v2_bff_cognito_client_secret: "${ secrets.getValue('account-portal-v2-bff-cognito_client_secret-awsdev') }",
  account_portal_v2_bff_express_session_secret: "${ secrets.getValue('account-portal-v2-bff-express_session_secret-awsdev') }",
  account_portal_v2_bff_password: "${ secrets.getValue('account-portal-v2-bff-password-awsdev') }",
  account_portal_v2_bff_token: "${ secrets.getValue('account-portal-v2-bff-token-awsdev') }",
  account_portal_v2_bff_username: "${ secrets.getValue('account-portal-v2-bff-username-awsdev') }",
  account_portal_v2_jwt_private_key: "${ secrets.getValue('account-portal-v2-jwt_private_key-awsdev') }",
  account_portal_v2_jwt_public_key: "${ secrets.getValue('account-portal-v2-jwt_public_key-awsdev') }",
  account_portal_v2_password: "${ secrets.getValue('account-portal-v2-password-awsdev') }",
  account_portal_v2_token: "${ secrets.getValue('account-portal-v2-token-awsdev') }  ",
  account_portal_v2_token_redo: "${ aws.getValue('/percipio/use1/develop/account-portal-v2-bff#token') }",
  account_portal_v2_invalid_key_1: "${ aws.getValue('/percipio/use1/develop/account-portal-v2-bff#invalid_key') }",
  account_portal_v2_invalid_key_2: "${ aws.getValue('DONT_EXIST') }",
  app_secret_1: "${ secrets.getValue('DONT_EXIST') }",
  nextgen_test_1: "<+ secrets.getValue('account-portal-v2-password-awsdev') >",
}
